src/ejabberd_ctl.erl

```diff
@@ -899,3 +900,16 @@ get_dist_proto() ->
         {ok, [Proto]} -> Proto;
         _ -> "inet_tcp"
     end.
+
+cets_info() ->
```

Comment:
This would need to be moved to a graphql handler, because we are moving away from `ejabberdctl`.